shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -643,6 +646,9 @@ - (void)deleteBackward {
 
   if (!_selectedTextRange.isEmpty)
     [self replaceRange:_selectedTextRange withText:@""];
+
+  // Backspace key code
+  [_keyEventDelegate performKeyPress:0x00000033 withCharacters:@""];
```

Comment:
Need your help on this, I feel meh about hard coding the constant but iOS doesn't explicitly list these constants anywhere. Also `0x00000033` maps to the delete button on macos.  An alternative is to map to the key code values seen by web browsers (iPad & iPhones). I wonder if Chromium has this listed somewhere, maybe we can pull it from there and auto gen the values, as we will need this on the framework which introduces a `RawKeyEventDataIOS` class to contain the data.

Reply:
Check out this, maybe it's what you're looking for? https://cs.chromium.org/codesearch/f/chromium/src/ui/events/keycodes/dom/keycode_converter_data.inc

Also take a look at the [gen_keycodes](https://github.com/flutter/flutter/tree/master/dev/tools/gen_keycodes) generator we use to generate the keycodes for various platforms. We keep a [database of key mappings](https://github.com/flutter/flutter/blob/master/dev/tools/gen_keycodes/data/key_data.json) too.

Reply:
According to this https://cs.chromium.org/chromium/src/remoting/client/input/native_device_keymap_ios.cc, we don't have access to the device keycode for iOS, and that they just use the USB key code.

I guess we have a choice on using the USB key code or the MacOS key code. Please advice. Here is the scan codes for the (delete key on iOS) from https://github.com/flutter/flutter/blob/master/dev/tools/gen_keycodes/data/key_data.json.
```
"scanCodes": {
      "android": [
        14
      ],
      "usb": 458794,
      "linux": 14,
      "xkb": 22,
      "windows": 14,
      "macos": 51
}
```

Reply:
Use the USB keycode.  The macOS keycode is extracted from the chromium source too, and if they're not using it, we probably shouldn't either.

Reply:
Alright, done.